cern_access/indico_cern_access/models/access_requests.py

```diff
@@ -40,16 +40,16 @@ class CERNAccessRequest(db.Model):
         nullable=False
     )
     birth_date = db.Column(
-        db.Date,
-        nullable=True
```

Comment:
Why are we removing that? I think it's good to indicate that we *want* nullable fields and didn't just forget to set it to False

Reply:
What happened to "nullable is already the default"? 😛

Reply:
I think it's an ugly default and I don't think I ever said that for this particular one :p

Reply:
Nullable is already the default in SQL, isn't it?

Reply:
yes :/